### ISSUE TITLE
Winappdriver fails to identify ui elements by Automation Id (Accessibility Id).

### ISSUE THREAD

**Issue by AkilanRagavaswamy** [microsoft/WinAppDriver#986]
Hi,
While trying to find elements by AutomationId using FindElementByAccessibilityId() method, im getting the following response in WinAppDriver console.

Console Output:

==========================================

POST /session/49F7855C-6214-4796-BB6E-E1529DA7581A/element/7.17796.49218346/click HTTP/1.1
Accept: application/json, image/png
Content-Length: 2
Content-Type: application/json;charset=utf-8
Host: 127.0.0.1:4723

{}
HTTP/1.1 200 OK
Content-Length: 63
Content-Type: application/json

{"sessionId":"49F7855C-6214-4796-BB6E-E1529DA7581A","status":0}

==========================================

POST /session/49F7855C-6214-4796-BB6E-E1529DA7581A/element HTTP/1.1
Accept: application/json, image/png
Content-Length: 49
Content-Type: application/json;charset=utf-8
Host: 127.0.0.1:4723

{"using":"accessibility id","value":"ExploreNow"}
HTTP/1.1 404 Not Found
Content-Length: 139
Content-Type: application/json

{"status":7,"value":{"error":"no such element","message":"An element could not be located on the page using the given search parameters."}}

Is there a work around for this, 
As i am running the automation in multiple resolutions, and the ui elements position is changing every time, finding by xPath (ui recorder code) needs to be changed every time i run the automation.
And also, finding by name cannot be used as, there are multiple buttons with same name in one view(screen).

Looking forward for your help and suggestions.
Thank You.

**Comment by samikshalambat** [comment]
If the element we need to identify or suppose click a button, but the button is not directly located it is on overlapping components the how to access such components????
